### ISSUE TITLE
Dropping air throws error

### ISSUE THREAD

**Issue by short509** [SkriptLang/Skript#2812]
### Description
error of function location

### Errors / Screenshots
![1](https://user-images.githubusercontent.com/27778635/74101114-9ddf3b00-4b79-11ea-823d-df26fc0f8714.png)
![2](https://user-images.githubusercontent.com/27778635/74101115-a0419500-4b79-11ea-9439-ae9632aae3af.png)
![3](https://user-images.githubusercontent.com/27778635/74101117-a0da2b80-4b79-11ea-9aa5-37d90ddf4aee.png)

Code:
```vb
function Stat_getCrops(p:player, b:block, l:location):
	set {_uuid} to uuid of {_p}
	chance of {Farmer::%{_uuid}%}%:
		if {_b} is melon block:
			set {_b} to melon slice
		play sound "BLOCK_BEEHIVE_SHEAR" with volume 1 and pitch 5 to {_p}
		set {_amount} to random integer between 2 and 4
		drop {_amount} of {_b} at {_l} # ◀error

on break of carrot plant:
	if "%block.getState().getData()%" contains "7":
		Stat_getCrops(player, event-block, event-block)
```

### Server Information
Skript version: 2.4.1
MC version: 1.15.2

**Comment by bluelhf** [comment]
Please share your code in ```code blocks```. This way we can figure out what's wrong, but looks like it's probably a problem with the script itself, not the plugin.

**Comment by ShaneBeee** [comment]
The issue is caused by attempting to drop air. This should be a relatively easy fix to prevent the error being thrown.

That said, Im not entirely sure why its attempting to drop air in this case.

**Comment by ShaneBeee** [comment]
Fixed in 2.5-alpha2
